packages/block-library/src/text-columns/editor.scss

```diff
@@ -3,3 +3,11 @@
 		outline: $border-width solid $gray-300;
 	}
 }
+
+// Hide inserter from previews.
+// This selector could be cleaner if we had a CSS class to target inside the preview iframe.
+html[style] body > .is-root-container .wp-block-columns {
```

Comment:
😅 Yeah, that's a weird stack of selectors. It would be way better if there were a more specific class we could tie into for the preview frame.

Reply:
It didn't feel right, so I just tried to push one that features a classname. But now I'd prefer a developer review of this one as well.

Reply:
👍 Yeah, this new one feels much better.